docs/general/community.md

```diff
@@ -33,6 +33,11 @@ contact and anyone doing so is likely trying to scam you.
 [Polkadot Meetup Platform](https://www.meetup.com/pro/polkadot) - Information on hosting meetups,
 applying for funding, and materials for running it.
 
+[Community Events](https://www.notion.so/web3foundation/Community-Events-Track-15aebfe73e934ff9991a5464ac10ba8e) -
```

Comment:
This link is not maintained. Do we have any website that tracks the latest events in the ecosystem?